docs/DEV_GUIDE.md

```diff
@@ -69,10 +69,29 @@ $ ./bin/ssvnode create-threshold --count <number of ssv nodes> --private-key <pr
 ```
 
 #### Generating an Operator Key
+To generate an operator key, you can use the `ssvnode generate-operator-keys command`. This command can generate the key in two ways: raw format or encrypted format.
```

Comment:
I also see this change in https://github.com/bloxapp/ssv/pull/1019, I think it should not be in this PR